manifest.yml

```
@@ -1,5 +1,5 @@
 manifest_version: 1
 name: SimpleIndexer
-description: A simple indexer that leverages the DocumentArrayMemmap
+description: A simple indexer that leverages DocumentArray
```

```suggestion
description: A simple indexer that leverages the built in SQLite storage backend of DocumentArray  
```

I think that we should do something about the sqlite backend otherwise people could think that it is only indexing in memory